target/linux/ath79/dts/ar7161_adtran_bsap1800v2.dts

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar7161_adtran_bsap1880.dtsi"
+
+/ {
+	model = "Adtran/Bluesocket BSAP-1800v2";
+	compatible = "adtran,bsap1800v2", "qca,ar7161";
```

Comment:
Just saw this: If v2 is a revision and not really included in the device's name, the proper way would be:
´´´
	model = "Adtran/Bluesocket BSAP-1800 v2";
	compatible = "adtran,bsap1800-v2", "qca,ar7161";
´´´
... so with space and hyphen.

This would obviously require updates in the other files